big_tests/tests/rest_client_SUITE.erl

```diff
@@ -38,7 +38,8 @@ all() ->
      {group, roster},
      {group, messages_with_props},
      {group, messages_with_thread},
-     {group, security}].
+     {group, security},
+     {group, sse}].
```

Comment:
Minor, just a suggestion: I would call it `sse_timeout` because it is timeout-specific, and there are other sse tests in this suite.